.github/workflows/build.yml

```diff
@@ -9,6 +9,7 @@ on:
   push:
     branches:
       - main
+      - multiarch-build
```

Comment:
Is it intended to have 2 different branches? I expected this would be part of main and all the builds would be multiarch. (and there'd be less maintainance to do)

Reply:
I will correct this.